components/sup/src/manager/mod.rs

```diff
@@ -59,29 +59,29 @@ use crate::{butterfly::{self,
                                  Suitability},
                         trace::Trace},
             common::{self,
```

Comment:
`common` and `butterfly` can also be broken out.

The `pub use crate::common::templating::package::Pkg;` a few lines above can also be tweaked.